x-pack/plugins/apm/public/components/app/ErrorGroupOverview/List/index.js

```diff
@@ -107,14 +106,15 @@ class List extends Component {
         render: (message, item) => {
           return (
             <MessageAndCulpritCell>
-              <TooltipOverlay content={message || NOT_AVAILABLE_LABEL}>
+              <EuiToolTip content={message || NOT_AVAILABLE_LABEL}>
                 <MessageLink hash={`/${serviceName}/errors/${item.groupId}`}>
                   {message || NOT_AVAILABLE_LABEL}
                 </MessageLink>
-              </TooltipOverlay>
-              <TooltipOverlay content={item.culprit || NOT_AVAILABLE_LABEL}>
+              </EuiToolTip>
+              <br />
```

Comment:
Never sure when to use `EuiSpacer`, `br` or margin. Have you got any prefs?

Reply:
Spacer is if you need space, IMO. This was just all on one line, so it was between this and wrapping both tooltips in <div>s, this seemed simpler.